nbs/examples/ElectricityLoadForecasting.ipynb

```diff
@@ -0,0 +1,3625 @@
+{
```

Comment:
<p>This should change with the correct <code>SeasonalNaive</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>